public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,84 @@
+# dvc.api.get_url()
+
+Return the URL
+([string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)
+type)) to the storage location of a <abbr>data artifact</abbr> tracked by DVC.
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Description
+
+Returns the full URL to the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target data artifact
+specified by its `path` in a `repo` (<abbr>DVC project</abbr>) is stored. The
+URL is formed by evaluating the corresponding
```

Comment:
evaluating -> extracting data from, reading, analyzing? evaluating has a special meaning in ML ... and sounds strange

Reply:
Sure, using `reading`.